apps/site/components/Downloads/Release/ReleaseCodeBox.tsx

```diff
@@ -1,5 +1,7 @@
 'use client';
 
+import createSval from '@node-core/mdx/evaluator';
```

Comment:
Not sure calling it /mdx makes sense.... I think making it clearer it is web stuff related makes sense? No idea for a name btw